spotify_player/src/client/mod.rs

```diff
@@ -345,6 +345,13 @@ impl Client {
             ClientRequest::DeleteFromLibrary(id) => {
                 self.delete_from_library(state, id).await?;
             }
+            ClientRequest::GetCurrentUserQueue => {
+                tracing::info!("Fetching current user queue");
```

Comment:
Nit: no need to add a log here. This kind of logging should already be handled by the caller function.